app/src/main/java/org/literacyapp/authentication/StudentImageCollectionActivity.java

```diff
@@ -63,10 +71,13 @@
     private AnimalOverlay animalOverlay;
     private MediaPlayer mediaPlayerInstruction;
     private MediaPlayer mediaPlayerAnimalSound;
+    private ImageView authenticationAnimation;
+    private boolean authenticationAnimationAlreadyPlayed;
 
     // Image collection parameters
     private static final boolean DIAGNOSE_MODE = true;
     private static final long TIMER_DIFF = 200;
+    private static long AUTHENTICATION_ANIMATION_TIME = 4000;
```

Comment:
This time is only for the StudentImageCollectionActivity. No TensorFlow is loaded here but the animation should get shown 3-4 seconds (workaround)